collections/_article/dev-snapshot-godot-4-4-dev-3.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "Dev snapshot: Godot 4.4 dev 3"
+excerpt: "We haven't slowed down and are bringing you another feature-packed update!"
+categories: ["pre-release"]
+author: Clay John
+image: /storage/blog/covers/dev-snapshot-godot-4-4-dev-3.webp
+image_caption_title: "TODO"
+image_caption_description: "TODO"
+date: 2024-10-03 16:00:00
+---
+
+We are back with another big dev update with over 330 commits merged in the last few weeks! Contributors continue to
+work on amazing new things while we continue to merge our backlog of important features that missed the 4.3 cutoff. The
+combined result is a lot of great stuff compressed into a short amount of time.
+
+Many of the changes in this release are bug fixes that will be backported to Godot 4.3 and released in 4.3.1! So please
+test this release well so we can be confident with the changes and release 4.3.1 with them as soon as possible.
+
+Keep in mind that while we try to make sure each dev snapshot is stable enough for general testing, this is by
+definition a pre-release piece of software. Be sure to make frequent backups, or use a version control system such as
+Git, to preserve your projects in case of corruption or data loss.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.4.dev2/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
```

Comment:
Can we add a link for the XR editor here as well. e.g:

You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.4.dev2/), [**XR editor**](https://www.meta.com/experiences/godot-game-engine/7713660705416473/) or the **Android editor** for this release.

Reply:
We can, though this whole paragraph will need rethinking eventually as it's a very lengthy blurb and I'm not sure how impactful it is. It might be better to feature all these in the download card directly.

BTW, how up to date are we on the XR editor and Android editor? Do you still manage to push new releases shortly after a dev snapshot is built? I've never figured out how to actually get the pre-release version of the Android editor from Google Play, so I feel we might want some dedicated instructions for this in the docs instead of bringing it along in each blog post.

Reply:
> BTW, how up to date are we on the XR editor and Android editor? Do you still manage to push new releases shortly after a dev snapshot is built?

Both versions are kept up-to-date, although for the XR editor, the dev releases are pushed directly to production given the user base is much smaller, whereas for the Android editor, the dev releases are pushed to internal and opt-in testing.

> I've never figured out how to actually get the pre-release version of the Android editor from Google Play, so I feel we might want some dedicated instructions for this in the docs instead of bringing it along in each blog post.

We have a page in the docs for the Android editor, where we could provide instructions for how to access the dev builds - https://docs.godotengine.org/en/stable/tutorials/editor/using_the_android_editor.html#limitations-known-issues
I'm planning to add a similar page for the XR editor.

Reply:
> I've never figured out how to actually get the pre-release version of the Android editor from Google Play, 

Well seems like I finally managed, looks like I had subscribed to the group with the wrong google account :P

Reply:
Done!